unit/analyses/constant_propagator.cpp

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************\
+
+Module: Unit test for constant propagation
+
+Author: Diffblue Ltd
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+
+#include <analyses/constant_propagator.h>
+
+#include <ansi-c/ansi_c_language.h>
+
+#include <goto-programs/goto_convert_functions.h>
+
+#include <util/message.h>
+
+#include <iostream>
```

Comment:
Why is iostream imported here?

Reply:
Because I'm silly, is why

Reply:
Wait no I'm not, this is used as the output for `goto_convert`

Reply:
Please don't. Unit tests have become very hard see failures in as they generate a ton of noise. Tests must be completely silent by default. If there is an error, then it should be reported, but not in any other case. As such: please report to a `std::ostringstream` and check that that output is empty.